package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pr-bumper",
-  "version": "1.11.2",
+  "version": "2.0.0",
```

Comment:
Manually bumping the version the last PR should have, but didn't b/c of the bug this PR fixes.